.github/workflows/tests.yml

```diff
@@ -104,7 +108,7 @@ jobs:
       matrix:
         os:
           - ubuntu-latest
-          - macos-latest
+          # - macos-latest
```

Comment:
What is the difference between `tests` and `pypy-tests` sections in workflows/tests.yml? Seems like `pypy-tests `section does not needed, because tests already runs in `tests` section.

Reply:
Separated because additional conditions in the job makes it harder to read and maintain, separated workflows easiest way to manage it.